server/testdriver/core/src/main/java/org/infinispan/server/test/core/persistence/ContainerDatabase.java

```diff
@@ -34,7 +37,17 @@
                      .env(env)
                      .build();
             });
-      container = new GenericContainer(image).withExposedPorts(port).waitingFor(Wait.forListeningPort());
+      container = new GenericContainer(image)
+            .withExposedPorts(port)
+            .withPrivilegedMode(true)
+            .waitingFor(Wait.forListeningPort());
+
+      String logMessageWaitStrategy = properties.getProperty("database.container.log.regex");
```

Comment:
The property should be documented and added to the `TestSystemPropertyNames` class

Reply:
@tristantarrant updated

Reply:
the PR is dirty, I will fix it